File: src/data/ListWeekDay.java
package data;

public class ListWeekDay extends ListDay {

  private Weekday weekday;

  public ListWeekDay(Appointment[] appointments, Therapist[] therapists, Weekday weekday) {
    super(appointments, therapists);
    this.weekday = weekday;
  }

  public Weekday getWeekday() {
    return weekday;
  }

  public void setWeekday(Weekday weekday) {
    this.weekday = weekday;
  }

}

File: frontend/src/class/ListWeekDay.ts
import Appointment from './Appointment';
import { Weekday } from './Enums';
import ListDay from './ListDay';
import Therapist from './Therapist';

export default class ListWeekDay extends ListDay {
  weekday : Weekday;

  constructor(
    appointments : Appointment[],
    therapists : Therapist[],
    weekday: Weekday,
  ) {
    super(appointments, therapists);
    this.weekday = weekday;
  }
}
